includes/jira.codehaus.org.inc

```diff
@@ -7,8 +7,9 @@
 ##################################
 # Owned by: Olivier Gaudin       #
 ##################################
-RewriteRule "^/browse/SONAR(.*)$" "http://jira.sonarsource.com/browse/SONAR$1" [L]
-RewriteRule "^/browse/MSONAR(.*)$" "http://jira.sonarsource.com/browse/MSONAR$1" [L]
+RewriteRule "^/browse/SONARSCTFS-(.*)$" "http://jira.sonarsource.com/browse/SONARTFVC-$1" [R=301,L]
```

Comment:
@hgomez-sonarsource I'm not sure this works as expected. `SONARSCTFS-4` should map to `SONARTFVC-11`

On Codehaus, we had 2 projects: SONARTFS and SONARSCTFS. All the tickets from SONARSCTFS got merged into the SONARTFS project - but this operation changed the ticket numbers. Jira was able to keep track of that.